deepchem/feat/tests/test_graph_data.py

```diff
@@ -129,3 +131,44 @@ def test_graph_data_single_atom_mol(self):
         assert str(
             graph
         ) == 'GraphData(node_features=[1, 32], edge_index=[2, 0], edge_features=None)'
+
+    @pytest.mark.torch
+    def test_graphdata_numpy_to_torch(self):
+        """
+        Test for converting GraphData numpy arrays to torch tensors
+        """
+        import torch
+        num_nodes, num_node_features = 5, 32
+        num_edges, num_edge_features = 6, 32
+        node_features = np.random.random_sample((num_nodes, num_node_features))
+        edge_features = np.random.random_sample((num_edges, num_edge_features))
+        edge_index = np.array([
+            [0, 1, 2, 2, 3, 4],
+            [1, 2, 0, 3, 4, 0],
+        ])
+        node_pos_features = None
+        # z is kwargs
+        z = np.random.random(5)
+
+        graph_np = GraphData(node_features=node_features,
+                             edge_index=edge_index,
+                             edge_features=edge_features,
+                             node_pos_features=node_pos_features,
+                             z=z)
+        graph = graph_np.numpy_to_torch()
+
+        assert isinstance(graph.node_features, torch.Tensor)
+        assert isinstance(graph.edge_index, torch.Tensor)
+        assert isinstance(graph.edge_features, torch.Tensor)
+        assert graph.node_pos_features is None
+        assert isinstance(graph.z, torch.Tensor)
+
+        batched_graph = BatchGraphData([graph_np, graph_np])
```

Comment:
Can we break this into 2 tests, one for `BatchGraphData` and one for `GraphData`?